### ISSUE TITLE
The default combination of search and matchesonscrollbar extensions doesn't work

### ISSUE THREAD

**Issue by Leonya** [codemirror/CodeMirror#5919]
Search matches are supposed to be highlighted on the scrollbar once the corresponding add-ons are loaded, without any configuration. Alas, it doesn't work anymore: https://codemirror.net/demo/search.html

**Comment by marijnh** [comment]
It does in both Firefox and Chrome Linux, so it'd be helpful if you include some information on the platform you are testing with.

**Comment by Leonya** [comment]
I have tried Chrome, Firefox, Safari on Mac: neither of them shows the markers

<img width="1552" alt="Screenshot 2019-06-27 at 22 51 41" src="https://user-images.githubusercontent.com/46537/60299967-3d72f480-992e-11e9-9859-ae4f8ef217c7.png">

**Comment by Leonya** [comment]
I have investigated the matter a bit more: the bug is caused by Mac's default "invisible scrollbars" behaviour. If I turn on the "Always show scrollbars" setting, the markers are shown.

**Comment by marijnh** [comment]
Right, it's basing the area where the markers can be shown on actual platform scrollbar width, so when the scrollbar isn't there, it doesn't show markers. I haven't looked into what other software that shows markers like this does in such a case.

**Comment by Leonya** [comment]
The thing is, this used to work before.

**Comment by marijnh** [comment]
In which version?

**Comment by Leonya** [comment]
I can't say for sure, but it did, 100%
